bin/merge_R1_umi.py

```diff
@@ -6,20 +6,24 @@
 parser = argparse.ArgumentParser()
 parser.add_argument("-R1", "--R1_filename", type=str, help="Path of R1 fastq.gz file.")
 parser.add_argument("-I1", "--I1_filename", type=str, help="Path of I1 fastq.gz file containing the UMI barcode.")
+parser.add_argument("-s", "--umi_start", type=int, help="UMI start position in index file.")
+parser.add_argument("-l", "--umi_length", type=int, help="UMI length.")
 parser.add_argument("-o", "--outname", type=str, default="UMI_R1.fastq.gz", help="Name of output file.")
 
 args = parser.parse_args()
 
 records_I1 = SeqIO.parse(gzopen(args.I1_filename, "rt"), format="fastq")
 records_R1 = SeqIO.parse(gzopen(args.R1_filename, "rt"), format="fastq")
+umi_start = args.umi_start
```

Comment:
You might be able to set this dependant on the umi_index sth. like:
```
if index_file then umi_start = 0
```
To avoid later problems

Reply:
Thanks for the comment, Leon, but this is not exactly the use case. The idea is that a Umi start different than 0, can only be set if the UMIs are provided in an index read file. And to make the user aware of that, I think the best is to throw an error if they try to set a UMI start, when an index read file is not provided.